docs/projects/cn0577/index.rst

```diff
@@ -0,0 +1,294 @@
+
+.. warning::
+    The VADJ for the Zedboard must be set to 2.5V.
+
+Configuration modes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - TWOLANES: specifies the number of lanes used
+
+    - 1 - two-lane output mode (default)
+    - 0 - one-lane output mode  
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The data path and clock domains are depicted in the below diagram:
+
+.. image:: ../cn0577/cn0577_zed_block_diagram.svg
+   :width: 800
+   :align: center
+   :alt: CN0577/ZedBoard block diagram
+
+Configuration modes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+tbc
+
+Jumper setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Depending on what configuration of pins is chosen on the jumpers P1, P2 and P3,
+the device can act in different modes, as described below. Of course, the PD
+jumper overrides the PD signal from the FPGA. It is controlled by a
+one-bit-adc-dac, in software.
+
+  - P1 - configures PD_N
+
+    - Shorting pins 1 and 2 → PD_N = 1, device is not powered down
+    - Shorting pins 2 and 3 → PD_N = 0, device is powered down
+  
+  - P2 - configures TESTPAT
+
+    - Shorting pins 1 and 2 → TESTPAT = 1, pattern testing is active
+    - Shorting pins 2 and 3 → TESTPAT = 0, pattern testing is inactive
+
+  - P3 - configures TWOLANES parameter
+
+    - Shorting pins 1 and 2 → TWOLANES = 1 (TWO LANES mode)
+    - Shorting pins 2 and 3 → TWOLANES = 0 (ONE LANE mode)
+
+The FMC connector connects to the LPC connector of the carrier board. For more
+detalis, check :adi:`EVALCN0577-FMCZ <CN0577>` datasheet.
+
+Clock scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  The clock architecture of the :adi:`EVALCN0577-FMCZ <CN0577>` is designed
+   with careful consideration to ensure low jitter and low phase noise
+
+-  An on-board 120 MHz voltage controlled crystal oscillator (VCXO) is used to
+   provide the clock for the :adi:`EVALCN0577-FMCZ <CN0577>` board and the FPGA.
+   It is further named as reference clock. This clock is gated and fed back to
+   the device as the sampling clock, on which the data was sampled
+
+-  The DMA runs on the ZynqPS clock FCLK_CLK0 which has a frequency of 100MHz
+
+For more detalis, check :adi:`EVALCN0577-FMCZ <CN0577>` datasheet.
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addresses are dependent on the architecture of the FPGA, having an offset
+added to the base address from HDL (see more at :ref:`architecture`).
+
+==================== ===============
+Instance             Zynq/Microblaze
+==================== ===============
+axi_ltc2387          0x44A0_0000
+axi_ltc2387_dma      0x44A3_0000
+axi_pwm_gen          0x44A6_0000
+==================== ===============
+
+I2C connections
```

Comment:
To be removed

Reply:
Again, followed CN0585 doc model I2C section. Should I remove from there too?

Reply:
This section should be removed because there are no project-specific I2C connections. These connections are specific to the carrier